bootstrap/helpers.php

```diff
@@ -0,0 +1,8 @@
+<?php
+
+use App\Helpers\LegislativeSessionHelper;
+
+function legislativeSessionHelper()
```

Comment:
.... ....       .............. Why????.....

Reply:
Lol, to bee more like how Laravel does it with `request('key')` or `auth()->user()`.

Reply:
If it's not Peter-ok'd then... removing it now.

Reply:
But those are there for specific reasons, namely so that they can be used in situations where the app doesn't already exist and Facades are not a thing. When would you ever face that with this method?